packages/create-audius-app/helpers/is-folder-empty.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
we going with kebab case for files?

Reply:
I am in favor of kebab case files generally because capitalization can cause git issues on windows. But in this case I just forked all this stuff from create-next-app